doc/user_guide/io.rst

```diff
@@ -625,15 +626,65 @@ as navigator in the plotting.
 
 .. _emd-format:
 
-EMD Electron Microscopy Datasets (HDF5)
----------------------------------------
+EMD
+---
 
 EMD stands for “Electron Microscopy Dataset.” It is a subset of the open source
 HDF5 wrapper format. N-dimensional data arrays of any standard type can be
-stored in an HDF5 file, as well as tags and other metadata. The EMD format was
-developed at Lawrence Berkeley National Lab (see http://emdatasets.com/ for
-more information). NOT to be confused with the FEI EMD format which was
-developed later and has a different structure.
+stored in an HDF5 file, as well as tags and other metadata. 
+
+EMD (NCEM)
+^^^^^^^^^^
+
+This EMD format was developed by Colin Ophus at the National Center for Electron Microscopy (NCEM). See http://emdatasets.com/ for more information.
+
+
+EMD (FEI)
+^^^^^^^^^
+
+This EMD format was developed by FEI for the Velox software. Although it shares similar structure than the other EMD format from NCEM, it differs in the way the data are stored. HyperSpy supports importing images, EDS spectrum and EDS spectrum image. For spectrum image, individual frame or individual EDS detector can be imported, however selecting these option will generate very large dataset. Therefore, the default is to import the sum over all frame and over all detectors. Alternatively, a specific frame range can be choosen -see the :ref:`Extra-loading-arguments-fei-emd` section below.
+Loading a spectrum image is slow if `numba <http://numba.pydata.org/>`_ is not installed.
+
+.. code-block:: python
+
+    >>> hs.load("sample.emd")
+    [<Signal2D, title: HAADF, dimensions: (|179, 161)>,
+    <EDSSEMSpectrum, title: EDS, dimensions: (179, 161|4096)>]
+
+.. warning::
+
+   This format is still not stable and files generated with the most recent version of Velox may not be supported.
+
+.. _Extra-loading-arguments-fei-emd:
+
+Extra loading arguments
++++++++++++++++++++++++
+
+- `load` : one of {'all', 'images', 'spectrums', 'spectrum_image'} (default is 'all')
```

Comment:
I would provide more details on what each of these keyword arguments do. At the moment the purpose of some parameters are describe in the intro, that of others after the list and that of some others not described at all.